lisa/sut_orchestrator/azure/credential.py

```diff
@@ -146,6 +144,19 @@ def type_name(cls) -> str:
     def type_schema(cls) -> Type[schema.TypedSchema]:
         return AzureCredentialSchema
 
+    def __init__(
```

Comment:
Please move it to a method, and call it in AzureDefaultCredential, ClientSecretCredential and the CertificationCredential. Because they depend on env variables.